### PR TITLE
Add 1.22.3 and 1.21.5 release notes

### PR DESCRIPTION
**Pull request by jaellio**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Review by zirain — APPROVED** [review]
LGTM

File: content/en/news/releases/1.22.x/announcing-1.22.3/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- **Updated** Go version to include security fixes for the net/http package related to [`CVE-2024-24791`](https://github.com/advisories/GHSA-hw49-2p59-3mhj)
+
+- **Updated** Envoy version to include security fixes related to [`CVE-2024-39305`](https://github.com/envoyproxy/envoy/security/advisories/GHSA-fp35-g349-h66f)
+
+- **Fixed** a bug where router's merged gateway was not immediately recomputed when a service was created or updated. ([Issue #51726](https://github.com/istio/istio/issues/51726)
+
+- **Fixed** inconsistent behavior with the `istio_agent_cert_expiry_seconds` metric.
+
+- **Removed** JSON access logs no longer emit keys in a stable ordering awaiting fix https://github.com/envoyproxy/envoy/issues/34420.
```
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.3/index.md:24]:
```suggestion
- **Removed** sorting of JSON access logs pending Envoy fix https://github.com/envoyproxy/envoy/issues/34420.
```

Comment (keithmattix) [reply content/en/news/releases/1.22.x/announcing-1.22.3/index.md]: Fixed with lints

File: content/en/news/releases/1.21.x/announcing-1.21.5/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Announcing Istio 1.21.5
+linktitle: 1.21.5
+subtitle: Patch Release
+description: Istio 1.21.5 patch release.
+publishdate: 2024-07-15
```
Comment (dhawton) [inline content/en/news/releases/1.21.x/announcing-1.21.5/index.md:6]:
```suggestion
publishdate: 2024-07-16
```

File: content/en/news/releases/1.22.x/announcing-1.22.3/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.22.3
+linktitle: 1.22.3
+subtitle: Patch Release
+description: Istio 1.22.3 patch release.
+publishdate: 2024-07-15
```
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.3/index.md:6]:
```suggestion
publishdate: 2024-07-16
```

**Review by dhawton — COMMENTED** [review]
I think we're targetting today to release.. if so, this should be it...

**Review by dhawton — APPROVED** [review]

**Pull request merged**